package.json

```diff
@@ -57,9 +57,9 @@
     "lint": "eslint src/ test/ example/ cli.js",
     "lint:fix": "npm run lint -- --fix",
     "pretest": "npm run lint && npm run build",
-    "unittest": "mocha -u tdd -r babelhook --recursive test/unit/ || true",
-    "unittest:bail": "mocha -u tdd -r babelhook --recursive test/unit/ --bail || true",
-    "test": "istanbul cover -root lib/ node_modules/mocha/bin/_mocha -- -u tdd -r babelhook --recursive test/unit/",
+    "unittest": "mocha -u bdd -r babelhook --recursive test/unit/ || true",
+    "unittest:bail": "mocha -u bdd -r babelhook --recursive test/unit/ --bail || true",
+    "test": "istanbul cover -root lib/ node_modules/mocha/bin/_mocha -- -u bdd -r babelhook --recursive test/unit/",
```

Comment:
What's the purpose of the change from `tdd` to `bdd`?

Reply:
the tdd specs does not include before_each but we use it.

Reply:
Understood. This change is fine then.